src/hydrusvideodeduplicator/dedup.py

```diff
@@ -314,21 +340,25 @@ def batched(iterable, n):
 
     # Check if files are trashed
     # Returns a dictionary of hash : trashed_or_not
-    def is_files_trashed_hydrus(self, file_hashes: list[str]) -> dict:
+    def is_files_trashed_hydrus(self, file_hashes: Iterable[str]) -> dict:
         videos_metadata = self.client.get_file_metadata(hashes=file_hashes, only_return_basic_information=False)[
             "metadata"
         ]
 
         result = {}
         for video_metadata in videos_metadata:
+            # This should never happen
+            if "hash" not in video_metadata:
+                logging.error("Hash not found for potentially trashed file.")
+                continue
             video_hash = video_metadata['hash']
-            is_trashed = video_metadata['is_trashed']
-            is_deleted = video_metadata['is_deleted']
+            is_trashed: bool = video_metadata.get('is_trashed', False)
```

Comment:
Ayyyy, I'd noticed this too. @_@ No idea why it happens either, but good catch!

Reply:
I don't know either. There's a lot of stuff in the API that makes me wonder. If I had more time I would really go through and make a bunch of PR's and set up some better testing.